gulpfile.js

```diff
@@ -31,6 +31,15 @@ builder.bundle('spec/testmodule.js', 'testmodule_3')
     .inDir('target/testmodule')
     .generateNoImportsBundle();
 
+//  - bundle "as" with a version - no prerelease tag 
+builder.bundle('spec/testmodule.js', 'testmodule@1.1.1')
```

Comment:
Yeah, the generated bundle has the version number on the end of it, but we do not want that to include prerelease tags if there are any there.